types/helpers.d.ts

```diff
@@ -1,12 +1,12 @@
-import Vue from 'vue';
+import { ComponentPublicInstance } from 'vue';
 import { Dispatch, Commit } from './index';
 
 type Computed = () => any;
 type InlineComputed<T extends Function> = T extends (...args: any[]) => infer R ? () => R : never
 type MutationMethod = (...args: any[]) => void;
 type ActionMethod = (...args: any[]) => Promise<any>;
 type InlineMethod<T extends (fn: any, ...args: any[]) => any> = T extends (fn: any, ...args: infer Args) => infer R ? (...args: Args) => R : never
-type CustomVue = Vue & Record<string, any>;
+type CustomVue = ComponentPublicInstance & Record<string, any>;
```

Comment:
Shall this change also fix this error? :heart: 

```
Cannot use namespace 'Vue' as a type.
     7 | type ActionMethod = (...args: any[]) => Promise<any>;
     8 | type InlineMethod<T extends (fn: any, ...args: any[]) => any> = T extends (fn: any, ...args: infer Args) => infer R ? (...args: Args) => R : never
  >  9 | type CustomVue = Vue & Record<string, any>;

```

Reply:
YES!